config/onebusaway-transit-data-federation-webapp-data-sources.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xmlns:context="http://www.springframework.org/schema/context"
+xsi:schemaLocation="
+    http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+
+    <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+        <property name="driverClassName" value="org.hsqldb.jdbcDriver" />
+        <property name="url" value="jdbc:hsqldb:file:/bundle/org_onebusaway_transit_data" />
```

Comment:
Is this right?

Reply:
I've always used `<property name="url" value="jdbc:hsqldb:file:/bundle/org_onebusaway_database" />` but if it seems to read from the database ok then it should work.

Reply:
In [onebusaway-transit-data-federation's application-context-common.xml](https://github.com/OneBusAway/onebusaway-application-modules/blob/8062291624aee26819f15176a9bba158914f9727/onebusaway-transit-data-federation/src/main/resources/org/onebusaway/transit_data_federation/application-context-common.xml#L42) it uses `${bundlePath}/org_onebusaway_transit_data` which is partly where I got this from. Also wasn't sure how that went along with config the bundle building uses.

Reply:
Ok, I'd stick with `org_onebusaway_transit_data` then.